.gitignore

```diff
@@ -1,6 +1,7 @@
 build*/
 exec*/
 install*/
+modulefiles/prepobs
```

Comment:
Is this a link that's created somewhere in the setup?  I don't see where it is created.